Source/WebCore/animation/ScrollTimeline.h

```diff
@@ -40,6 +40,12 @@ class RenderStyle;
 
 class ScrollTimeline : public AnimationTimeline {
 public:
+    struct ScrollTimelineData {
```

Comment:
You can make this `private` and rename to just `Data`.

Reply:
Fixed.